AUTHORS.md

```diff
@@ -11,52 +11,62 @@ contributions are subject to the project's copyright under the terms of the
 
 This file keeps track of authors contributions.
 
-## Maintainers / Steering Comittee
+## Maintainers / Steering committee
 
 ---
 
-- **Romain Hugonnet** [@rhugonnet](https://github.com/rhugonnet) <romain.hugonnet@gmail.com>
-- **Amaury Dehecq** [@adehecq](https://github.com/adehecq)
-- **Erik Schytt Mannerfelt** [@erikmannerfelt](https://github.com/erikmannerfelt)
-- **Emmanuel Dubois** [@duboise-cnes](https://github.com/duboise-cnes) <emmanuel.dubois@cnes.fr>
-- **Valentine Bellet** [@Valentine-Bellet](https://github.com/valentine-bellet) <valentine.bellet@cnes.fr>
-- **Alice de Bardonnèche-Richard** [@adebardo](https://github.com/adebardo) <alice.de-bardonneche-richard@cs-soprasteria.com>
-
+| Maintainer                       | GitHub                                     | Affiliation                    | Email                                   |
+|----------------------------------|--------------------------------------------|--------------------------------|-----------------------------------------|
+| **Romain Hugonnet**              | [@rhugonnet](https://github.com/rhugonnet) | University of Alaska Fairbanks | [📧](mailto:romain.hugonnet@gmail.com)  |
+| **Amaury Dehecq**                | [@adehecq](https://github.com/adehecq)     | Université Grenoble Alpes      | N/A                                     |
```

Comment:
Good idea to have the table!
One needs to add "French Research Institute for Sustainable Development (IRD)" to my affiliations.